crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -51,6 +51,12 @@ pub enum Error {
     #[snafu(display("Data sources [{}] does not exist", data_source.iter().map(TableReference::to_quoted_string).join(", ")))]
     DataSourcesNotFound { data_source: Vec<TableReference> },
 
+    #[snafu(display("Failed to retrive table {} data source", data_source.to_quoted_string()))]
```

Comment:
```suggestion
    #[snafu(display("Failed to retrieve table {} data source", data_source.to_quoted_string()))]
```

I don't think this error message conforms with the new guidelines, @peasee to weigh in.

Reply:
Updated error message.

> Failed to find table '{table_name}'. An internal error occurred during vector search.\nPlease report a bug on GitHub: https://github.com/spiceai/spiceai/issues"